### ISSUE TITLE
Admin chat feature

### ISSUE THREAD

**Issue by frdmn** [CircuitSoftGroup/SlackMC#13]
Hello,

I'm not sure if this is generally wanted, but I would love to use your plugin as kind of an admin chat:

In the configuration I would specify another Slack channel which is used as "admin chat" and ingame there would be a new command (`/admin <msg>` or whatever) to send messages into this channel.

And all messages sent from that given channel out of Slack show up for the people with the permission `slack.admin`.

Let me know what do you think about this.

**Comment by mastercoms** [comment]
I think this is outside the scope of this plugin and would be more suited for another plugin to hook into this one's API.

**Comment by frdmn** [comment]
@mastercoms Yeah, I already thought so. Unfortunately my development skills are not that great (sys admin in full time) so I can't do that. :/ 

Perhaps someone wants this as badly as I do and is indeed able to write an own plugin for this. 

Thanks for your quick response, though!

**Comment by mastercoms** [comment]
I'll write a plugin for you. No one will use my API except for me anyway :)

**Comment by frdmn** [comment]
@mastercoms Woah! That would be really awesome. Let me know if you need any more information. I do my best to help you out during the development of the plugin! :astonished:

**Comment by mastercoms** [comment]
Well, I'm sorry, but until 1.5.x is released and stable, I won't be able to work on the admin chat plugin.

**Comment by frdmn** [comment]
@mastercoms No worries! I'm already happy because of your offer :)

Regarding the name: I'm not quite sure, perhaps something simple like "SlackAdminChat"?

**Comment by mastercoms** [comment]
Your idea gave me a new idea. SlackMC channels. So you can have multiple webhook URLs and then a plugin using the API can specify a certain channel. Kind of like permission groups.

**Comment by frdmn** [comment]
That would be even more perfect!
